R/path.R

```diff
@@ -113,7 +116,7 @@ standardise_connection <- function(con) {
   con
 }
 
-standardise_one_path <- function(path, write = FALSE, call = caller_env()) {
+connection_or_filepath <- function(path, write = FALSE, call = caller_env()) {
```

Comment:
From its name, it's easy to expect that `standardise_path()` is (approximately) `map(several_inputs, standardise_one_path)` and that is just really not the case.

I'm renaming standardise_one_path() to better reflect what it does.